web3/web3/method.py

```diff
@@ -0,0 +1,127 @@
+import utils.utils
+import errors
+
+
+class Method:
```

Comment:
This is only necessary because we want to support Python 2 as well, correct? (Because in Python 3, _object_ is inherited from implicitly if it is omitted). http://stackoverflow.com/questions/4015417/python-class-inherits-object/9448136#9448136